src/main/scala/codacy/codenarc/CodeNarcOutput.scala

```diff
@@ -15,7 +16,7 @@ object CodeNarcOutput {
         case violation: Violation =>
           CodeNarcOutput(
             fileResults.getPath,
-            violation.getMessage,
+            Option(violation.getMessage).getOrElse(violation.getRule.getName),
             violation.getRule.getName,
```

Comment:
Can you please name a variable before like `val ruleName = violation.getRule.getName` and use that ?